minimal/staging.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:8005:80"
\ No newline at end of file
+      - "0.0.0.0:8005:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
```

Comment:
In that case, we'll need to have separate `.env` files too. That way we can have a valid certificate for `staging.domain.com` and another for production URL.

I'll update that and also include instructions in the README.